src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -251,6 +263,112 @@ def _get_missing_data(
     )
 
 
+def reconcile_aggregates(
+    ws: WorkspaceClient,
+    spark: SparkSession,
+    table_recon: TableRecon,
+    reconcile_config: ReconcileConfig,
+    local_test_run: bool = False,
+):
+    """[EXPERIMENTAL] Reconcile the aggregated data between the source and target tables.
+    for e.g., COUNT, SUM, AVG of columns between source and target with or without any specific key/group by columns
+    Supported Aggregate functions: MIN, MAX, COUNT, SUM, AVG, MEAN, MODE, PERCENTILE, STDDEV, VARIANCE, MEDIAN
+    """
+    # verify the workspace client and add proper product and version details
+    # TODO For now we are utilising the
+    #  verify_workspace_client from transpile/execute.py file. Later verify_workspace_client function has to be
+    #  refactored
+
+    ws_client: WorkspaceClient = verify_workspace_client(ws)
+
+    report_type = ""
+    logger.info(f"report_type: {report_type}, data_source: {reconcile_config.data_source}")
```

Comment:
Report type is empty in the line above.

Reply:
Yes, it's intentional. As report_type is not applicable for Aggregate Data Reconciliation.  

For initializing Reconciliation and ReconCapture classes, we need to pass report_type. If I change to report_type to None, there is a cascading effect and it needs refactoring.  

Avoiding refactoring in this PR, focusing on only feature implementation.

Reply:
My concern is that we are printing the report type in the log, which will always be blank, so it serves no purpose. Could we use a more suitable message?

Reply:
Yes, updated the `logger` to print `report_type`, only when it is available.